cpu/mips32r2_common/Makefile.include

```diff
@@ -10,13 +6,36 @@ export MEMORY_BASE=0x80000000
 export MEMORY_SIZE=1M
 export APP_START=0x80000000
 
-include $(MIPS_ELF_ROOT)/share/mips/rules/mipshal.mk
+# Portable 'lowercase' func.
```

Comment:
While this looks definitely beautiful, we're using ```$(shell echo X | tr 'A-Z' 'a-z')``` in many other places. Mental note to myself: analyze what is faster and then unify.